integrations/box-js/entrypoint.sh

```diff
@@ -1,5 +1,5 @@
 #!/bin/sh
-
+mkdir -p ${LOG_PATH}
```

Comment:
not required, the LOG_PATH gets created in the main IntelOwl dockerfile.

Reply:
Oh, you are right